Analysis/output/country_data_report.Rmd

```diff
@@ -1,12 +1,19 @@
 ---
-title: ""
+title: "Data and Model Outputs Report"
 output: html_document
 params:
-  cholera_directory: "~/projects/cholera-mapping-pipeline/"
-  config: "Analysis/configs/2015_2019_country/config_SSD_2015_2019.yml"
+  cholera_directory: "~/gh/cholera-mapping-pipeline/"
```

Comment:
can we have this default to a relative filepath instead of an absolute computer specific file path